core/consensus/synchronizer/impl/synchronizer_impl.cpp

```diff
@@ -30,21 +38,31 @@ namespace kagome::consensus {
 
   void SynchronizerImpl::blocksRequest(
```

Comment:
Actually requestBlocks would be a more canonical name for a method, as it contains a verb.